tests/performance/C-molecule/README

```diff
@@ -0,0 +1,93 @@
+Carbon molecules QMC Performance Benchmarks
+
+1. Introduction
+
+These benchmarks are VMC and DMC runs for carbon molecules
+in order to measure LCAO performance for various system sizes.   
+This is in contrast to the conventional integration tests where
+the particle counts are too small to be representative.
+Care is still needed to remove initialization, I/O, and compute
+a representative performance measure.
+
+See the explanation, discussion, and disclaimer in ../NiO/README for
+general guidelines. If you are interested in setting up a new benchmark
+or figure of merit, we encourage you to contact the developers.
+
+The ctest integration is sufficient to run the benchmarks and measure
+relative performance from version to version of QMCPACK and assess
+proposed code changes. To obtain highest performance on a particular
+platform, you will have to run the benchmarks in a standalone manner
+and tune thread counts, placement, walker count (etc.)
+
+2. Simulated system and QMC methods tested
+
+The simulated systems are various sizes of a carbon ring (C12,C18,C24,
+and C30) and C60 buckyball. All-electron tests (ae) using the cusp
+correction and pseudopotential tests (pp) are included.
+
+Name  Atoms Type  Electrons  Electrons per spin
+ C12    12   AE       72         36
+ C12    12   PP       48         24
+ C18    18   AE      108         54
+ C18    18   PP       72         36
+ C24    24   AE      144         72
+ C24    24   PP       96         48
+ C30    30   AE      180         90
+ C30    30   PP      120         60
+ C60    60   PP      240        120
+
+Runs consist of a number of short blocks of (i) VMC without drift (ii)
+VMC with drift term included (iii) DMC with constant population.
+
+Two sets of wavefunction are tested: LCAO with a one and
+two body Jastrow functions, and a more complex form with an additional
+three body Jastrow function. 
+
+The tests require at least QMCPACK 3.6.0.
+
+3. Requirements
+
+Download the necessary C-molecule h5 orbital files of different sizes from
+the following link
+
+https://anl.box.com/s/pveyyzrc2wuvg5tmxjzzwxeo561vh3r0
```

Comment:
This is the correct link.
https://anl.app.box.com/s/88z6quep9e7gp8vqagtupkg73zdumpaj

Reply:
Change of strategy. Single URL for all files from now on.